src/context.ts

```diff
@@ -1,17 +1,10 @@
 import { createContext } from 'react';
 
 export interface Config {
-  speed: number
-  color: string
-  colors: string[]
 }
 
 export function configDefaults() {
-  return {
-    speed: 2000,
-    color: '#afa',
-    colors: ['#aaffaa', '#ffaaaa', '#aaaaff']
-  };
+  return {};
```

Comment:
does it make sense to leave them here if these are just empty obje?cts

Reply:
Will clean up unused files in a separate PR.